iOS/Flows/Conversation/Cells/TimeMachineCollectionViewLayout.swift

```diff
@@ -67,13 +76,16 @@ class TimeMachineCollectionViewLayout: UICollectionViewLayout {
     var zPosition: CGFloat {
         return self.collectionView?.contentOffset.y ?? 0
     }
+    /// The focus position of the last item in the collection, as determined by sort value.
     var maxZPosition: CGFloat {
         return self.itemFocusPositions.values.max() ?? 0
     }
     /// A cache of item layout attributes so they don't have to be recalculated.
     private var cellLayoutAttributes: [IndexPath : UICollectionViewLayoutAttributes] = [:]
     /// A dictionary of z positions where each item is considered in focus. This means the item is frontmost, most recent, and unscaled.
     private(set) var itemFocusPositions: [IndexPath : CGFloat] = [:]
+    /// A cache of all the sort values for each item.
+    private(set) var itemSortValues: [IndexPath : Double] = [:]
```

Comment:
Storing the sort values is useful for maintaining the content offset when more items are loaded.